File: eshop.client/src/models/review/review.ts
import { Category } from "../category/category";
import { Supplier } from "../supplier/supplier";
import { User } from '../user/user';
export class Review {
  id?: number;
  userId!: number;
  rating!: number;
  text!: string;
  productId!: number;
  userName?: string;
}

File: Eshop.Server/Models/Review.cs
﻿namespace Eshop.Server.Models
{
    public class Review
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int Rating { get; set; }
        public string Text { get; set; }

        public int ProductId { get; set; }

        public Product Product { get; set; } = null!;
        public User User { get; set; } = null!;
    }
}
